app/src/main/res/values/strings.xml

```diff
@@ -11,6 +11,7 @@
     <string name="closing_msg">Closing\u2026</string>
     <string name="opening_msg">Opening\u2026</string>
     <string name="merging_msg">Merging\u2026</string>
+    <string name="reviewing_msg">Adding Review\u2026</string>
```

Comment:
@maniac103 Unrelated to this pull request but we should probably use the actual 3 dots character instead of this code.

Reply:
What would it change? This *is* the 3 dot character, just encoded.

Reply:
Make it clear for these who'll look here in future what these are without the need to check Unicode.